ISISComputingGroup/ibex_gui
# Ticket5069

### Description of work

Add configuresBlockGWAndArchiver tag to the client because of the way the block server saves the config (clears all fields of its config and repopulates the fields with what the client has sent). This way the client receives the tag and sends its back but we can't edit it from the gui (something we could add later if we wanted).

To test:

- Create a config with the tag configuresBlockGWAndArchiver as true
- Ensure it is loaded as the current config
- Check the client log to see that the tag configuresBlockGWAndArchiver is set as true in the client
- Edit the current configuration (you don't have to make any changes)
- Save the config
- Check the configuresBlockGWAndArchiver tag is still true in the meta.xml of the config

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/5069

### Acceptance criteria

*List the acceptance criteria for the PR. The aim is provide information to help the reviewer*

### Unit tests

*Give an overview of unit tests you have added or modified, if applicable. The aim is provide information to help the reviewer*

### System tests

*Mention any automated tests or manual tests that you have added or modified, if applicable. The aim is provide information to help the reviewer*

### Documentation
*Highlight and provide a link to any additions or changes to the documentation, if applicable. The aim is provide information to help the reviewer*

---

#### Code Review


### Final Steps
